src/clj/game/core/props.clj

```diff
@@ -16,8 +16,7 @@
    (let [updated-card (update! state side (update card prop-type #(+ (or % 0) n)))
          args {:counter-type prop-type :amount n :placed placed}]
      (if (= prop-type :advance-counter)
-       (do (when (and (ice? updated-card)
-                      (rezzed? updated-card))
+       (do (when ((every-pred ice? rezzed?) updated-card)
```

Comment:
Mind reverting this? `every-pred` is clever and correct but inconsistent with the rest of the app. Maybe if it wasn't so long and awkward lol.

Reply:
Done